doc/importing-template.md

```diff
@@ -0,0 +1,69 @@
+## How to import the template into Azure DevOps
+
+### Pre-requisites:
+
+[Create a DevOps Organizaton](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops#create-an-organization) or use an existing one. 
+
+### Process:
+
+1. Navigate to the [Azure DevOps Demo Generator](https://azuredevopsdemogenerator.azurewebsites.net/Account/Verify) and sign into your DevOps organization.
+2. Go to https://aka.ms/fta/iacmigrationtemplate, sign into your DevOps organization and you already will be in the correct template selection for IaC Migration (Please note that if you click on Choose template, this template is available under **FastTrack for Azure** tab as well)
+3. Select **Create Project** on the main page. 
+4. After the successfully provisioning of the template, go to the Project > Boards > Backlogs and on the top-right click on the engine (Configure team settings). Validate that Epics and Stories are selected and then click **Save and Close**. 
+
+![backlog-setting-0](/png/backlog-setting-0.png)
+
+
+![backlog-setting-1](/png/backlog-setting-1.png)
+
+5. Now change to view the Epics then you have all set, containing all Epics, User Stories and Tasks.
+
+![backlog-setting-2](/png/backlog-setting-2.png)
+
+### Customization:
+
+It is recommended to utilize Wave, Deliverable, and Activity terminology in naming the work items, so here is a guidance over the process of customization to have something different than Epics, Stories and Tasks:
+
+1. Click on the Azure DevOps logo in the top 
+2. Click on Organization Settings
+3. Go to Boards > Process
+4. Right click on Agile then click to Create inherited process
+5. Name as you want. In this case let's name as Migration, then click to Create process
+6. Click on the Process created (Migration in this case)
+7. On Work item types, click to **+ New work item type** and let's create three: Wave, Deliverable, and Activity.  Please note that you can customize the icon and the color for each one. After you click to create, a new window will be expanded to set extra details as Layout, States and Rules. You don't need change nothing here, just return to previous screen and continue the creation.
```

Comment:
Last sentence suggestion: Nothing needs to be modified on this screen, you can return to previous screen (`Migration Process` at the top of the page) and continue the creation of the other items.

Reply:
Done